prime-router/src/main/kotlin/azure/BlobAccess.kt

```diff
@@ -234,6 +242,75 @@ class BlobAccess() : Logging {
             val previousBlobItemVersions: List<BlobItem>?,
         )
 
+        enum class SchemaType {
+            FHIR_FHIR,
+            FHIR_HL7,
+        }
+
+        /**
+         * Downloads the content of each blob in the [directory] with [blobContainerInfo] takes the input file, runs
+         * the transform on it, and compares it to the output validating that they are the same.
+         */
+        fun validateSchemas(
```

Comment:
This seems to be one of a few upcoming functions directly related to the "Manage Translation Schemas" feature. It would be good to move this function into another class (along with the future ones) since these functions do not implement "BlobAccess" but rather make use of it.